marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -190,6 +190,7 @@ const Flows: React.FC<Props> = (props) => {
             onOk={() => onOk(flowName)}
             onCancel={() => onCancel()}
             width={350}
+            destroyOnClose={true}
```

Comment:
@wooldridge @briantang we need to add these to all the modal in future PRs so they dont lie around hidden in the DOM. It affects the tests since labels like Yes/No or Cancel/Save is common between them.

Reply:
@bsrikan Makes sense. Should I create a task for adding this property in to all Modals across the app?